orc8r/cloud/helm/orc8r/charts/secrets/Chart.yaml

```diff
@@ -13,7 +13,7 @@ apiVersion: v1
 appVersion: "1.0"
 description: A Helm chart to create magma orchestrator secrets
 name: secrets
-version: 0.1.10
```

Comment:
Throughout: you'll likely have to update the versions based on latest master. Do that last, once we get through the rest of the review process